server/tm_cmds.go

```diff
@@ -49,27 +47,19 @@ func ShowValidatorCmd() *cobra.Command {
 			cfg := serverCtx.Config
 
 			privValidator := pvm.LoadOrGenFilePV(cfg.PrivValidatorKeyFile(), cfg.PrivValidatorStateFile())
-			valPubKey, err := privValidator.GetPubKey()
+			pk, err := privValidator.GetPubKey()
 			if err != nil {
 				return err
 			}
-
-			output, _ := cmd.Flags().GetString(cli.OutputFlag)
-			if strings.ToLower(output) == "json" {
-				return printlnJSON(valPubKey)
-			}
-
-			pubkey, err := sdk.Bech32ifyPubKey(sdk.Bech32PubKeyTypeConsPub, valPubKey)
+			out, err := codec.ProtoMarshalJSONI(pk, nil)
 			if err != nil {
 				return err
 			}
-
-			fmt.Println(pubkey)
+			fmt.Println(string(out))
```

Comment:
@aaronc , can you confirm if this makes sense?
+ we use the JSON protobuf marshaler for pubkey
+ shall we convert to string - as it is done here?

Reply:
Showing the pubkey Any as JSON, as you did in the latest commits, sgtm 👍